week-2/G-conditionals-2/exercise-2.js

```diff
@@ -7,8 +7,12 @@
 
 */
 
-function studentPassed(grade) {
-
+function studentPassed(grade1) {
+  if (grade1 < 50) {
```

Comment:
👍 There is also a convenience ternary operator that allows coding this in this into one line
`return grade1 < 50 ? "passed" : "failed";`

condition ? <value for true> : <value for false>;